### ISSUE TITLE
Starter Block Placement Bug is back

### ISSUE THREAD

**Issue by Tisdag1** [tsunamayo/Starship-EVO#5914]
### Short Description of the issue:

That starter block placement bug is back, where it snaps to a different angle than the direction you rotate it to before placing it.
It seems to be happening in survival only. I have started a few new games in both survival & creative and it only happens in survival.

### Steps to reproduce the issue:

Create a starter block and rotate it to different angles. It will keep snapping to a different angle when placed.

### Environment
- Starship EVO version:  24w30b
- Graphics Card: 

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Before Placing
![20240727154850_1](https://github.com/user-attachments/assets/aa1e2627-4013-4495-b07b-09eacf40309f)

After Placing
![20240727154902_1](https://github.com/user-attachments/assets/1fba1442-1654-4967-9bb4-a3842b1e19e3)

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

[Player.log](https://github.com/user-attachments/files/16401550/Player.log)

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by tsunamayo** [comment]
@Tisdag1 Hi, I tried to reproduce but cant, it is working fine for me. Are you doing something special? Are you able to reproduce straight after loading the game? Thanks

**Comment by Tisdag1** [comment]
Nope, I wasn't doing anything special. Just created several new survival games, harvested scrap from the wreckage. Built the starter block, rotated it to my desired angle & tried placing it. 
  It's the strangest thing. Last night I was constantly having the problem on the new games I created to test it (even after reloading). And now for the life of me, I can't get it to act up again. In the handful of test games I created today to duplicate it, they all worked fine & placed in any direction I rotated them to.
  It did happen once, but was fixed upon reloading the test game.
